### PR TITLE
Fizzbuzz

### PR DESCRIPTION
**Pull request by mjgpy3**
Write fizzbuzz!

Proves that dytype is interview-complete!

### CODE REVIEW

**Comment by mjgpy3** [comment]
@Kazark on a more serious note, what's something I could help with?

**Comment by Kazark** [comment]
"Interview-complete": I like it!

File: dytype/dytype.scm [hunk 1]
```diff
@@ -110,9 +112,13 @@
 (define prelude
   '(
     (+ . (* . 1))
+    (modulo . 3)
+    (number->string . 2)
     (cons . 3)
     (list . (* . 1))
+    (iota . 4)
```
Comment (Kazark) [inline dytype/dytype.scm:119]: I'm not familiar with `iota`. I guess it's your enumerator?

**Review by Kazark — COMMENTED** [review]
I totally think we should merge these changes.

**Comment by Kazark** [comment]
What's your reasoning for not merging? I think this is great.

**Comment by mjgpy3** [comment]
No good reason.

File: dytype/dytype.scm (hunk 1 above)
Comment (mjgpy3) [reply dytype/dytype.scm:119]: Yeah, it's a lot like python2's `range` but with the arguments all over the place.

Comment (mjgpy3) [reply dytype/dytype.scm:119]: According to the docs the name is from APL, haha!

**Pull request merged**